src/io/output_time.cc

```diff
@@ -212,6 +256,9 @@ void OutputTime::write_time_frame()
 			// Remember the last time of writing to output stream
 			write_time = time;
 			current_step++;
+            
+            // unset validity of the output mesh after data are written at the current write time
+            is_output_mesh_valid_ = false;
```

Comment:
OK, you're right.
I did not deleted the mesh in the right place...
So now since it is shared pointer, I hope "output_mesh_.reset()" is safe, isn't it?